docs: Add documentation for loan write off (#180)
frappe/erpnext_documentation: erpnext_documentation/www/docs/user/manual/en/loan-management/loan-write-off.md

<!-- add-breadcrumbs -->
# Loan Write Off
> Introduced in Version 13

**Loan Write Off is used to write off the pending principal amount against a loan in order to close the loan.**

> To access the Loan Interest Accrual list, go to:

Reviewer: ```suggestion
To access the Loan Write Off list, go to:
```

> Home > Loan Management > Disbursement and Repayment > Loan Write Off

> <img class="screenshot" alt="Loan Interest Accrual" src="{{docs_base_url}}/assets/img/loan-management/loan-write-off.png">

Reviewer: ```suggestion
<img class="screenshot" alt="Loan Write Off" src="{{docs_base_url}}/assets/img/loan-management/loan-write-off.png">
```

## 1. Prerequisites
Before creating a Loan Write Off, you must create the following first:

* [Loan](/docs/user/manual/en/loan-management/loan)

> ## 2. How to Create a Loan Write Off

Reviewer: ```suggestion
## 2. How to create a Loan Write Off
```

> 1. Go to Loan Write Off List, click on New.

Reviewer: ```suggestion
1. Go to the Loan Write Off list, click on New.
```

> 2. Select Loan for which the principal amount has to be written off. The Applicant Type and Applicant will be fetched from the Loan.

Reviewer: ```suggestion
2. Select the Loan for which the principal amount has to be written off. The Applicant Type and Applicant will be fetched from the Loan.
```

3. Enter Posting Date.

> 4. Select Write Off Account.

Reviewer: ```suggestion
4. Select the Write Off Account.
```

> 5. Enter write off amount.

Reviewer: ```suggestion
5. Enter the Write Off Amount.
```
